### PR TITLE
Update README

### LINKED ISSUE #160: Update README
The README file doesn't reflect latest changes. It would nice to build this file for testing.

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Closes #160 by updating the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,7 +189,7 @@ The Feature Labs open source community is happy to provide support to users of C
 4. For everything else, the core developers can be reached by email at [help@featurelabs.com](mailto:help@featurelabs.com).
```
Comment (jeff-hernandez) [inline README.md:189]: Our we still using help@featurelabs.com?

File: README.md [hunk 2]
```diff
@@ -330,7 +210,7 @@ Compose open source has been developed by Feature Labs engineering team. The ope
 
 ## Feature Labs
 <a href="https://www.featurelabs.com/">
-    <img src="http://www.featurelabs.com/wp-content/uploads/2017/12/logo.png" alt="Featuretools" />
+    <img src="https://www.featurelabs.com/wp-content/uploads/2017/12/logo.png" alt="Featuretools" />
 </a>
 
 Compose is an open source project created by Feature Labs. We developed Compose to enable flexible definition of the machine learning task. Read more about our rationale behind automating and developing this stage of the machine learning process here.
```
Comment (jeff-hernandez) [inline README.md:216]: I'm assuming we need to update this to Innovation Labs.

File: README.md (hunk 1 above)
Comment (kmax12) [reply README.md:189]: nah, i think we can just delete this bullet. they should use the other channels

File: README.md [hunk 3]
```diff
@@ -1,44 +1,51 @@
+<p align="center"><img width=50% src="docs/source/images/compose.png" alt="Compose" /></p>
```
Comment (jeff-hernandez) [inline README.md:1]: To render on PyPI, we'll need to switch back to absolute paths for the images before merging to main.

```
https://raw.githubusercontent.com/FeatureLabs/compose/main/docs/source/images/compose.png
```

**Review by rwedge — DISMISSED** [review]
Looks good

File: README.md (hunk 2 above)
Comment (rwedge) [reply README.md]: @kmax12 thoughts?

**Pull request merged**